ftw/simplelayout/browser/resources/integration.theme.scss

```diff
@@ -294,17 +294,23 @@ ul[class^="sl-toolbar"] {
 }
 
 .simplelayout-initialized body {
-  padding-right: $editbar-height;
+  @include screen-small() {
+    padding-right: $editbar-height;
+  }
 }
 
 .sl-toolbox {
   @include noselect();
+  @include screen-small() {
+    display: table;
```

Comment:
@maethu according to http://caniuse.com/#feat=css-table there are no issues with IE11 concerning table layout